doc: tcm: remove toolchain manager app (#21034)
nrfconnect/sdk-nrf: doc/nrf/dev_model_and_contributions/adding_code.rst

* Follow the instructions in the :ref:`dm-wf-get-ncs` section of the documentation.
  This requires you to install Git and west, but you can then ignore them from that point onwards.

> If you need to update the copy of the |NCS| you are working with, you can obtain the source code again, or, if you have kept the original set of repositories, update it by following steps in :ref:`dm-wf-update-ncs`.

Reviewer: ```suggestion
If you need to update the copy of the |NCS| you are working with, you can obtain the source code again, or, if you have kept the original set of repositories, follow the steps in :ref:`dm-wf-update-ncs` to update it.
```

This approach also allows you to fork additional |NCS| repositories and point to those.
This can be useful if you have to make changes to those repositories beyond adding your own application to the manifest repository.

> With this approach, first get the source code (for example using the :ref:`command line <dm-wf-get-ncs>`), and then :ref:`fork the sdk-nrf repository <dm-wf-fork>`.

Reviewer 2: ```suggestion
With this approach, first get the source code (for example, using the :ref:`command line <dm-wf-get-ncs>`), and then :ref:`fork the sdk-nrf repository <dm-wf-fork>`.
```

> In order to use this approach, you first need to get the source code (for example using :ref:`command line <dm-wf-get-ncs>`), and then :ref:`fork the sdk-nrf repository <dm-wf-fork>`.

Reviewer: ```suggestion
With this approach, first get the source code (for example using the :ref:`command line <dm-wf-get-ncs>`), and then :ref:`fork the sdk-nrf repository <dm-wf-fork>`.
```

* `Recommended: Using the example application repository`_
* `Creating your own manifest repository from scratch`_

> Once you have your new manifest repository hosted online, you can use it with west just like you use the `sdk-nrf`_  repository when getting the source code (for example using the :ref:`command line <dm-wf-get-ncs>`) and later :ref:`updating it <dm-wf-update-ncs>`.

Reviewer 2: ```suggestion
Once you have your new manifest repository hosted online, you can use it with west the same way you use the `sdk-nrf`_  repository when getting the source code (for example, with the :ref:`command line <dm-wf-get-ncs>`) and later :ref:`updating it <dm-wf-update-ncs>`.
```

> Once you have your new manifest repository hosted online, you can use it with west the same way you use the `sdk-nrf`_  repository when getting the source code (for example, with the :ref:`command line <dm-wf-get-ncs>`) and later :ref:`updating it <dm-wf-update-ncs>`.

Reviewer 3: ```suggestion
Once you have your new manifest repository hosted online, you can use it with west the same way you use the `sdk-nrf`_ repository when getting the source code (for example, with the :ref:`command line <dm-wf-get-ncs>`) and later :ref:`updating it <dm-wf-update-ncs>`.
```
